_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,405 @@
+---
+layout:       element
+title:        Forms
+date:         2015-05-12 16:02:35
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- forms
+
+slug:         forms
+scssurl:      "https://github.com/edx/ux-pattern-library/blob/master/_src/edx-pl/sass/components/_forms.scss"
+
+description:  These examples demonstrate how to use form controls within your application.  We've grouped the form controls by type and for each type, provided proper CSS classes to achieve the desired visual presentation. In some cases, we've created optional classes which enhance the presentation. All of the examples have been tested to be accessible. Theming is available as well so form controls can match your institutions look and feel.
+
+info: Forms allow users to interact with the interface, set preferences, and enter any necessary personal information. They also form the base of individual exercises within each class.
+---
+
+<h3 class="hd-4 example-set-hd">Forms</h3>
+<h4 class="hd-6 example-set-hd">Text inputs</h4>
+<div class="example-set">
+    <fieldset class="field-group">
+        <legend class="field-group-hd sr-only">Personal Details</legend>
+        <div class="field">
```

Comment:
Please remove all required attributes/characteristics from this example. You have a required example below.